e2e-tests/e2e-legacy/sparql-editor/saved-query/readonly-query.spec.js

```diff
@@ -32,12 +26,12 @@ describe.skip('Readonly saved query', () => {
     });
 
     afterEach(() => {
-        LoginSteps.logout();
-        cy.deleteRepository(repositoryId);
-        UserAndAccessSteps.visit();
-        LoginSteps.loginWithUser(USER_ADMINISTRATOR, PASSWORD);
-        UserAndAccessSteps.toggleSecurity();
-        cy.deleteUser(USER_NAME);
+        cy.loginAsAdmin().then(()=> {
+            return cy.switchOffSecurity(true);
```

Comment:
How handling these commands in the promise chain helps more  than the built-in mechanism for automatic waiting in cypress?!